src/main/java/org/spongepowered/common/service/scheduler/SchedulerBase.java

```diff
@@ -52,14 +52,14 @@ protected String nextName(PluginContainer plugin) {
 
     /**
      * Gets the timestamp to update the timestamp of a task. This method is task
-     * sensitive to support different timestamp types i.e. wall clock and ticks.
+     * sensitive to support different timestamp types i.e. nanotime and ticks.
      *
      * @param task The task
      * @return Timestamp for the task
      */
     protected long getTimestamp(ScheduledTask task) {
-        // Supports wall clock time by default
-        return System.currentTimeMillis();
+        // Supports nanotime by default
```

Comment:
The base scheduling implementation supports wall clock time (seconds/milliseconds/nanoseconds/whatever). The synchronous scheduler adds an additional timestamp which is the number of ticks.

You don't need to change the comment, it's just saying that a system clock time is supported